File: Homework1Csharp/Homework1Csharp/Form1.cs
namespace Homework1Csharp
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void label1_Click(object sender, EventArgs e)
        {

        }

        private void button1_Click(object sender, EventArgs e)
        {
            Displaylabel.Text = "Hello, I am '" + txt_name.Text + "', I am from '" + txt_from.Text + "'";
        }

        private void label3_Click(object sender, EventArgs e)
        {

        }
    }
}

File: Homework1VB/Homework1VB/Form1.vb
Public Class Form1
    Private Sub Label1_Click(sender As Object, e As EventArgs) Handles Label1.Click

    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        displaylabel.Text = "Hello, I am '" + txt_name.Text + "', I am from '" + txt_from.Text + "'"
    End Sub

    Private Sub RichTextBox1_TextChanged(sender As Object, e As EventArgs) Handles txt_name.TextChanged
    End Sub

    Private Sub Label3_Click(sender As Object, e As EventArgs) Handles displaylabel.Click

    End Sub
End Class
